bot_sample.go

```diff
@@ -74,20 +72,11 @@ func main() {
 
 	webSocketClient.Listen()
 
-	ticker := time.NewTicker(5 * time.Second)
-	defer ticker.Stop()
-
 	go func() {
 		for {
```

Comment:
We can further simplify this by just ranging the channel instead:

```go
for resp := range client.EventChannel {
  handle(resp)
}
```

Reply:
Good catch! I updated the func() to use this syntax